doc/whats_new/v1.1.rst

```diff
@@ -589,6 +589,10 @@ Changelog
 :mod:`sklearn.multiclass`
 .........................
 
+- |Enhancement| :class:`sklearn.multiclass.OneVsOneClassifier` now have a cached_property
+  `class_pairs_` storing classes for which each estimator is trained with.
+  :pr:`22436` by :user:`Diwakar-Gupta`.
```

Comment:
```suggestion
  :pr:`22549` by :user:`Diwakar-Gupta`.
```
The changelog check is failing because your PR number should be referenced here.